static/js/components/ExpandedLearningResourceDisplay_test.js

```diff
@@ -375,19 +375,22 @@ describe("ExpandedLearningResourceDisplay", () => {
         object.items.length
       )
     })
-
-    it(`should include a display of list items for ${objectType}`, () => {
-      const object = makeLearningResource(objectType)
-      const wrapper = render({
-        object
-      })
-      assert.ok(wrapper.find(".expanded-learning-resource-userlist").exists())
-      R.zip(
-        object.items.map(item => item.content_data),
-        wrapper.find("LearningResourceRow").map(el => el.prop("object"))
-      ).forEach(([obj1, obj2]) => assert.deepEqual(obj1, obj2))
-    })
   })
+  ;[LR_TYPE_USERLIST, LR_TYPE_LEARNINGPATH, LR_TYPE_PROGRAM].forEach(
```

Comment:
Can you include a test or condition for a different type where the user list should not be rendered?